overview pages for BTC staking / timestamping (#35)
babylonchain/babylonchain.github.io: docs/introduction/btc-timestamping/overview.md

# Bitcoin Timestamping

Babylon's Bitcoin timestamping Protocol introduces an innovative approach to timestamping PoS blockchains on Bitcoin, enhancing their integrity and security.

> With Bitcoin's unparalleled security, these timestamps on Bitcoin can be used to resolve various security issues with the PoS blockchain, such as the long range attacks.

Reviewer: ```suggestion
With Bitcoin's unparalleled security, these timestamps on Bitcoin can be used to resolve various security issues with the PoS blockchain, such as long range attacks.
```

> The protocol includes Bitcoin as the timestamping service, the babylon blockchain as the checkpoint aggregation and data availability service, and other PoS blockchains as the consumers of security.

Reviewer: ```suggestion
The protocol includes Bitcoin as the timestamping service, the Babylon blockchain as the checkpoint aggregation and data availability service, and other PoS blockchains as the consumers of security.
```

> Bitcoin timestamping services as a key building block for the [Bitcoin staking](/docs/introduction/btc-staking/overview.md) protocol, and provides various use cases, including:

Reviewer: ```suggestion
Bitcoin timestamping serves as a key building block for the [Bitcoin staking](/docs/introduction/btc-staking/overview.md) protocol, and provides various use cases, including:
```

> - **Fast Unbonding:** PoS blockchains require social consensus to circumvent long range attacks and this leads to long unbonding periods. Bitcoin security replaces social consensus and reduces unbonding periods to 5 hours.

Reviewer: ```suggestion
- **Fast Unbonding:** PoS blockchains require social consensus to circumvent long range attacks and this leads to long unbonding periods. Bitcoin security replaces social consensus and reduces unbonding periods to a few hours.
```
